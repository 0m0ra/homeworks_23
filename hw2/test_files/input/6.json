{
    "Kathy Wright": {
        "region": "Moscow",
        "registered": "2023-05-03",
        "last_login": "2023-05-11",
        "email": "nicholas99@example.org",
        "age": 22
    },
    "Ricardo Schmidt": {
        "region": "London",
        "registered": "2005-07-08",
        "last_login": "2023-09-04",
        "email": "john81@example.net",
        "age": 52
    },
    "Jim Horne": {
        "region": "Sochi",
        "registered": "2003-08-15",
        "last_login": "2023-07-20",
        "email": "dfinley@example.com",
        "age": 46
    },
    "Clayton Webster": {
        "region": "Paris",
        "registered": "2021-10-04",
        "last_login": "2023-08-13",
        "email": "kevinrogers@example.net",
        "age": 57
    },
    "Patrick Smith": {
        "region": "New-York",
        "registered": "2013-05-03",
        "last_login": "2023-08-04",
        "email": "qcole@example.org",
        "age": 12
    },
    "Natasha Hamilton": {
        "region": "Moscow",
        "registered": "2003-10-21",
        "last_login": "2023-04-06",
        "email": "dennisrobert@example.org",
        "age": 60
    },
    "Ryan Baker": {
        "region": "Paris",
        "registered": "2003-10-18",
        "last_login": "2023-04-15",
        "email": "nbaker@example.org",
        "age": 45
    },
    "Tiffany Romero": {
        "region": "London",
        "registered": "2017-05-18",
        "last_login": "2023-06-10",
        "email": "acostalarry@example.org",
        "age": 60
    },
    "John Kennedy": {
        "region": "London",
        "registered": "2015-04-07",
        "last_login": "2023-05-20",
        "email": "mrobinson@example.org",
        "age": 71
    }
}