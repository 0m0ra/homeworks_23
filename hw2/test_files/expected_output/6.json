{
    "max_age": 71,
    "min_age": 12,
    "avg_age": 47,
    "median_age": 52,
    "lt_two_days_offline_users_average_age": 0,
    "lt_week_offline_users_average_age": 0,
    "lt_month_offline_users_average_age": 0,
    "lt_half_of_year_offline_users_average_age": 45,
    "mt_half_of_year_offline_users_average_age": 52
}